modules/migration-configuring-mcg.adoc

```diff
@@ -0,0 +1,171 @@
+// Module included in the following assemblies:
+//
+// migration/migrating_3_4/configuring-replication-repository.adoc
+// migration/migrating_4_1_4/configuring-replication-repository.adoc
+// migration/migrating_4_2_4/configuring-replication-repository.adoc
+[id='migration-configuring-mcg_{context}']
+= Configuring a Multi-Cloud Object Gateway storage bucket as a replication repository
+
+You can install the OpenShift Container Storage Operator and configure a Multi-Cloud Object Gateway (MCG) storage bucket as a replication repository on the CAM tool cluster.
```

Comment:
I remember it requires OCS 4.3. Should it mention OCS exact version?

Reply:
No. I'll make sure the branch is not cherry-picked for 4.2. Will update PR description. Thanks.